relay/relay.py

```diff
@@ -56,6 +56,7 @@ def start(self):
         self._load_config()
         self._load_contracts()
         self._load_orderbook()
+        logger.info('using RPCProvider with parameters %s' % self.config['rpc'])
```

Comment:
Lets please use the newer .format for string formatting instead of %.

Reply:
Or do you have arguments for using %?

Reply:
It's what I'm used to and it's what is being used in C/JAVA. But I'm also fine with using .format.